docs/iot/sample/iot-sample.md

```diff
@@ -0,0 +1,538 @@
+# Network Audio Sample Overview
+
+The Network Audio sample application demonstrates how the user can manage things. The source code shows work with the st-things SDK in use cases for initialization, starting, ownership transfer, and resource handling.
+
+## Prerequisites
+
+-   To ensure proper application execution, the following privileges must be set:
+    -   `http://tizen.org/privilege/network.get`
+    -   `http://tizen.org/privilege/network.set`
+    -   `http://tizen.org/privilege/internet`
+    -   `http://tizen.org/privilege/alarm.set`
+    -   `http://tizen.org/privilege/network.profile`
+-   Tizen Studio 2.0 (or later) must be installed.
+-   In the Package Manager, 4.0 Mobile in the **Main SDK** must be installed.
+-   In the Package Manager, **Extras &gt; IOT-Headless-4.0** in the **Extension SDK** must be installed.
+-   IoT Setup Wizard (Tizen Studio plugin) must be installed.
+-   The device must be flashed through an SD card on the IoT Setup Wizard, and connectivity drivers must be installed.
+-   You must prepare the certificate (`*.pem`) and private key (`*.der`).
```

Comment:
Recommendation : This string is modified as below on TD.
Certificate (`*.pem`) and private key (`*.der`) must be prepared.